.github/FUNDING.yml

```
@@ -1 +1,12 @@
+# These are supported funding model platforms
+
+github: # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
+patreon: # Replace with a single Patreon username
+open_collective: # Replace with a single Open Collective username
+ko_fi: # Replace with a single Ko-fi username
+tidelift: # Replace with a single Tidelift platform-name/package-name e.g., npm/babel
+community_bridge: # Replace with a single Community Bridge project-name e.g., cloud-foundry
 liberapay: TeamNewPipe
+issuehunt: # Replace with a single IssueHunt username
+otechie: # Replace with a single Otechie username
```

Do we have to have those other lines? They're not needed.
```suggestion
liberapay: TeamNewPipe
```